ffs/api/api.go

```diff
@@ -360,3 +420,21 @@ func (i *API) Close() error {
 	i.closed = true
 	return nil
 }
+
+func (i *API) ensureValidColdCfg(cfg ffs.ColdConfig) error {
```

Comment:
This is used to check that if cold config is enabled, that the specified address is actually one managed by this instance